card/lib/card/mod/dirs.rb

```diff
@@ -17,12 +17,20 @@ def initialize mod_paths
         end
       end
 
+      # add a new mod to mod load paths
       def mod mod_name
+        if @mods.include? mod_name
+          raise Error,
+                "name conflict: mod with name \"#{mod_name}\" already loaded"
+        end
         @mods << mod_name
-        # TODO: do something if two mods have the same name?
         @paths[mod_name] = File.join @current_path, mod_name
       end
 
+      def path mod_name
+        @paths[mod_name]
+      end
```

Comment:
not sure about this api with #mod and #path.   maybe it should be add_path and path?